httpclient-backend/src/main/scala/sttp/client/httpclient/package.scala

```diff
@@ -0,0 +1,17 @@
+package sttp.client
+
+import java.nio.ByteBuffer
+
+package object httpclient {
```

Comment:
maybe move to sttp.client.internal? There's some bytebuffer-related util funcitons already